maltoolbox/language/languagegraph.py

```diff
@@ -256,14 +338,23 @@ class LanguageGraphAttackStep:
     type: str
     asset: LanguageGraphAsset
     ttc: dict = field(default_factory = lambda: {})
+    overrides: bool = False
     children: dict = field(default_factory = lambda: {})
     parents: dict = field(default_factory = lambda: {})
-    description: dict = field(default_factory = lambda: {})
-    attributes: Optional[dict] = None
+    info: dict = field(default_factory = lambda: {})
+    inherits: Optional[LanguageGraphAttackStep] = None
+    tags: set = field(default_factory = lambda: {})
+    _attributes: Optional[dict] = None
```

Comment:
Why do you keep reachable steps and requirements in `_attributes` and not directly as LanguageGraphAttackStep attributes, i.e. `step.reaches` and `step.requires`?